### PR TITLE
criu: move timers dump/restore code into separate file

### PR DESCRIPTION
**Pull request by ccccrrrr**
Fixes: #335

move timers dump/restore code into separate file

### CODE REVIEW

File: criu/include/timer.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#ifndef __ITIMER_H__
+#define __ITIMER_H__
+
+#include "types.h"
+#include "parasite.h"
```
Comment (avagin) [inline criu/include/timer.h:5]: why do we need to include these two headers here?

File: criu/include/timer.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#ifndef __CR_TIMER_H__
+#define __CR_TIMER_H__
+
+#include "restorer.h"
```
Comment (avagin) [inline criu/include/timer.h:4]: why do we need to include restorer.h?

File: criu/include/timer.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+extern int prepare_itimers(int pid, struct task_restore_args *args, CoreEntry *core);
```
Comment (avagin) [inline criu/include/timer.h:6]: We need to include a header that declares CoreEntry.

Comment (ccccrrrr) [reply criu/include/timer.h:6]: thanks for your tip, my timer.h is not very well written

**Comment by avagin** [comment]
Merged, thanks.

**Pull request closed without merging**